app/controllers/votes_controller.rb

```diff
@@ -13,6 +13,9 @@ def show
   # GET /votes/new
   def new
     @vote = Vote.new
+    # buildでデータベースに登録せずにフォームの作成
+    @vote.choices.build
+    @vote.choices.build
```

Comment:
[want]
https://github.com/chikusui-ryo/sample-app/blob/8f42271ed19edacf809da044311ed77f7006e234/app/models/choice.rb#L1-L3
に以下に標準の選択肢の数を占めす定数を設定して
```ruby
DEFAULT_CHOICES = 2
```


```suggestion
    DEFAULT_CHOICES.times.each { @vote.choices.build }
```
このようにループ処理で実装するのはいかがでしょうか？